src/assets/scripts/client/utilities/windUtilities.js

```diff
@@ -0,0 +1,18 @@
+import { getNormalDistributedNumber } from '../math/core';
+
+export const getNewWind = (data) => {
+    const speed = getNormalDistributedNumber(data.speed);
+    const initialAngle = getNormalDistributedNumber(data.angle);
+    let newAngle = initialAngle;
+
+    if(initialAngle > 360) {
+        newAngle = initialAngle - 360;
```

Comment:
what happens if this receives `750`?  is that even possible?  If so, what happens?

Reply:
we have a helper function for this actually, `degrees_normalize()` I think it's called.